lib/6to5/transformation/transformers/optional-external-runtime.js

```diff
@@ -0,0 +1,11 @@
+exports.optional = true;
+
+// In theory, it would be more appropriate to do this in `manipulateOptions`,
+// but we need an identifier for the import and we can't get that before the
+// AST is built.
+exports.ast = {
+  enter: function (ast, file) {
+    file.opts.runtime = file.addImport(file.opts.runtime, "to5Runtime").name;
+    file.dynamicImports[file.dynamicImports.length - 1]._noInteropRequire = true;
```

Comment:
`addImport` already does that.  My concern is it conflicting with other dynamic imports e.g. `core-js/library` such that it ends up after these.